setup.py

```diff
@@ -44,6 +44,7 @@
 Programming Language :: Python :: 3.5
 Programming Language :: Python :: 3.6
 Programming Language :: Python :: 3.7
+Programming Language :: Python :: 3 :: Only
```

Comment:
Do we need both this line and line 43? Can you point to the "HOWTO" documentation for these specifiers?